### DISCUSSION TITLE
New nuget.org Beta?

### DISCUSSION THREAD

**Discussion by odalet** [SixLabors/ImageSharp.Drawing#199] [General] [answered]
Hi!

Now that **SixLabors.ImageSharp** nuget is v2.0.0 👍, any plan to publish new 'nuget.org' betas of **SixLabors.Fonts** and **SixLabors.ImageSharp.Drawing** built against it?

**Comment by tocsoft** [comment, marked as answer]
As we announced in our blogpost the plan was to release them shortly but we are not willing to promise anything concreate at this time.
https://sixlabors.com/posts/announcing-imagesharp-200/#the-future

**Reply by odalet** [reply]
Thanks for the reply: I wasn't aware of the blog post. And I get it, better a quality release than a quick one!

**Reply by JimBobSquarePants** [reply]
There’s a new beta out today. Enjoy!

**Reply by odalet** [reply]
Great! Gonna download it right now!
